src/Shouldly.Tests/ConventionTests/ApprovePublicApi.ShouldlyApi.approved.cs

```diff
@@ -330,6 +351,47 @@ public class ShouldContainWithCountShouldlyMessage : Shouldly.ShouldlyMessage
     {
         public ShouldContainWithCountShouldlyMessage(object? expected, object? actual, int matchCount, string? customMessage, [System.Runtime.CompilerServices.CallerMemberName] string shouldlyMethod = null) { }
     }
+    public class ShouldMatchApprovedException : Shouldly.ShouldAssertException
+    {
+        public ShouldMatchApprovedException(string? message, string? receivedFile, string? approvedFile, bool viewerNotPresent) { }
```

Comment:
`bool viewerNotPresent` added.  Can add prior constructor for non-breaking change if desired.